packages/@storefront/structure/src/filtered-list/index.ts

```diff
@@ -26,13 +26,17 @@ class FilteredList {
   }
 
   onKeyDown(event) {
```

Comment:
Noted, will do.

Reply:
@Vic-Dev Since the `onKeyDown` method expects a `KeyboardEvent`, but the `onClick` method which it may invoke expects a `MouseEvent`, this required some 'funky' casting. Please review.

Reply:
@Vic-Dev Do you have any thoughts on this?

Reply:
Since @jmbtutor has taken over review of this PR, I'm going to mark this as resolved.